README.md

```diff
@@ -313,6 +313,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [restlet.com](https://restlet.com/products/apispark/) — APISpark enables any API, application or data owner to become an API provider in minutes via an intuitive browser interface.
   * [Roboflow](https://roboflow.com) - create and deploy a custom computer vision model with no prior machine learning experience required. Free tier includes up to 1,000 free source images.
   * [ROBOHASH](https://robohash.org/) - Web service to generate unique (cool :) images from any text.
+  * [SalesforceToNotion](https://salesforcetonotion.com/) - SaaS tool that integrates data between Salesforce and Notion. With this add-on, you can pull Salesforce data directly into your Notion workspace.
```

Comment:
I dont understand if this is really a SaaS it seems to be a chrome addon rather than a service?

Reply:
Yes, that's correct.

Reply:
Sorry, I saw a similar tool in the list and thought it's good idea to add my tools.

Reply:
please let us know which other are pure chrome extensions with no SaaS component, or send a PR to remove them.